frontend/components/panels/BaseClassPanel.js

```diff
@@ -59,6 +60,7 @@ class BaseClassPanel extends React.Component {
 
   onReqClick(reqType, childBranch, event) {
     // Create the React element and add it to retVal
+    // TODO: Do we need this anymore?
```

Comment:
Yup, I think we do. I just tried adding a `return;` at the beginning of this function and that causes the page to reload every time you click a requisite link. This function prevents the page reload (`event.preventDefault();`) and then fires off the new search without reloading the page.

Reply:
Alright, documented!